docs/api/system-preferences.md

```diff
@@ -106,6 +106,15 @@ This is necessary for events such as `NSUserDefaultsDidChangeNotification`.
 
 Same as `unsubscribeNotification`, but removes the subscriber from `NSNotificationCenter`.
 
+### `systemPreferences.registerDefaults(defaults)` _macOS_
+
+* `defaults` Object - a dictionary of (`key: value`) user defaults
+  * `key` String - must be of type `string`
```

Comment:
> String - must be of type `string`

Is it really necessary to say "string" twice? )
And technically it can be anything, the value will be automatically converted to a string.